### ISSUE TITLE
loss nan

### ISSUE THREAD

**Issue by Tiandishihua** [ZiqiaoPeng/SyncTalk#60]
I used the dataset you provided for training and found that the loss diverged.

**Comment by wning13** [comment]
我的经验是在[这里](https://github.com/ZiqiaoPeng/SyncTalk/blob/4293c63d40fba8dfc8122de94d5e8b55eac93341/nerf_triplane/network.py#L382) `exp`前个`clamp`会有帮助

**Comment by Tiandishihua** [comment]
> 我的经验是在[这里](https://github.com/ZiqiaoPeng/SyncTalk/blob/4293c63d40fba8dfc8122de94d5e8b55eac93341/nerf_triplane/network.py#L382) `exp`前个`clamp`会有帮助

非常感谢您的指定和帮助

**Comment by Tiandishihua** [comment]
> 我的经验是在[这里](https://github.com/ZiqiaoPeng/SyncTalk/blob/4293c63d40fba8dfc8122de94d5e8b55eac93341/nerf_triplane/network.py#L382) `exp`前个`clamp`会有帮助

非常感谢您的指点和帮助

**Comment by schxnhxlz** [comment]
> 我的经验是在[这里](https://github.com/ZiqiaoPeng/SyncTalk/blob/4293c63d40fba8dfc8122de94d5e8b55eac93341/nerf_triplane/network.py#L382) `exp`前个`clamp`会有帮助

Hi there, where did you change that? I have the same issue.

**Comment by einsqing** [comment]
@wning13 这样调整么？
`sigma = torch.clamp(torch.exp(h[..., 0]))`

**Comment by Jason-cs18** [comment]
> @wning13 这样调整么？ `sigma = torch.clamp(torch.exp(h[..., 0]))`

Do you have find a solution? In my custom datasets, I always meet this issue. I have tried to use a smaller learning rate (`--lr` and 'lr-net'), but `loss=Nan` still exists.

**Comment by wning13** [comment]
> torch.exp

```
sigma = torch.exp(torch.clamp(h[..., 0], -50.0, 50.0))
```
我用了fp32，使用fp16的话，范围要改小一点，大概是[-9, 9]

**Comment by wennjiee** [comment]
> > torch.exp
> 
> ```
> sigma = torch.exp(torch.clamp(h[..., 0], -50.0, 50.0))
> ```
> 
> 我用了fp32，使用fp16的话，范围要改小一点，大概是[-9, 9]

你好，我想请问下这个区间是怎么确定的呢？

**Comment by ZiqiaoPeng** [comment]
This bug has been fixed.
